dashboard/views.py

```diff
@@ -110,6 +114,52 @@ def file_access(request):
 
     return HttpResponse(df.to_json(orient='records'))
 
+
+def grade_distribution(request):
+
+    # Later this could be coming from a table specific to the course
+    bins = [0, 50, 65, 78, 89, 100]
```

Comment:
That's a good idea, I'm sure each course is going to want these to be configurable. This will probably have to pull from Canvas as we incorporate more of the canvas API's? 

https://canvas.instructure.com/doc/api/grading_standards.html#method.grading_standards_api.context_index

Reply:
This needs some research, I don't know if courses have the grading schema published in canvas or not. Some might have it in canvas but not sure if they totally follow while grading